webpack-build-utils/terminal.js

```diff
@@ -0,0 +1,13 @@
+const { exec } = require('child_process');
+
+function runInTerminal() {
+    exec(process.argv[2], error => {
+        if (error) {
+            console.error(`exec error: ${error}`);
```

Comment:
Shouldn't that be `${error.toString()}` or else you might end up with `exec error: [Object]`.

Reply:
Good point